frontend/src/container/Trace/Filters/Panel/PanelBody/index.tsx

```diff
@@ -25,7 +26,7 @@ function PanelBody(props: PanelBodyProps): JSX.Element {
 
 	return (
 		<Card bordered={false}>
-			{type === 'duration' ? <Duration /> : <CommonCheckBox name={type} />}
+			{type === 'duration' ? <Duration /> : type==='traceID' ? <TraceID />: <CommonCheckBox name={type} />}
```

Comment:
Could a type and its corresponding render component be passed in from props themselves and get rid of nested ternary op?